### PR TITLE
1.3.0

### PR DESCRIPTION
**Pull request by signalnerve**
https://github.com/issues?utf8=%E2%9C%93&q=repo%3Acloudflare%2Fwrangler+merged%3A%3E%3D2019-08-16

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,195 @@
...
+
+    With multiple environments defined, `wrangler build`, `wrangler preview`, and `wrangler publish` now accept a `--env` flag to indicate what environment you'd like to use, for instance, `wrangler publish --env production`.
+
+    To support developers transitioning to environments, we've written documentation for the feature, including further information about deprecations and advanced usage. [Check out the documentation here!](https://github.com/cloudflare/wrangler/blob/master/docs/environments.md)
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/385]: https://github.com/cloudflare/wrangler/issues/385
+    [pull/386]: https://github.com/cloudflare/wrangler/pull/386
+
+  - **KV commands - [ashleymichal], [issue/339][pull/405]**
```
Comment (ashleymichal) [inline CHANGELOG.md]: if we can list more than one contributor, i think we should put gabbi here, she wrote A LOT of this code.
```suggestion
  - **KV commands - [ashleymichal, gabbifish], [issue/339][pull/405]**
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,195 @@
...
+
+    Wrangler 1.3.0 includes commands for managing and updating [Workers KV](https://www.cloudflare.com/products/workers-kv/) namespaces, keys, and values directly from the CLI.
+
+    - `wrangler kv:namespace`
+
+      `wrangler kv:namespace` allows developers to `create`, `list`, and `delete` KV namespaces, from the CLI. This allows Wrangler users to configure new namespaces and add them directly into `wrangler.toml` without having to navigate into the Cloudflare Web UI to manage namespaces. Neat!
```
Comment (ashleymichal) [inline CHANGELOG.md]: this makes it sound like we will add the namespace to your toml for you, and that is not the case. I know the example output says 'add the following...', but we should make the language consistent.

File: CHANGELOG.md (hunk 1 above)
Comment (signalnerve) [reply CHANGELOG.md]: ah totally! my little keyboard shortcut only grabs the PR creator. good call!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,196 @@
...
+    [issue/385]: https://github.com/cloudflare/wrangler/issues/385
+    [pull/386]: https://github.com/cloudflare/wrangler/pull/386
+
+  - **KV commands - [ashleymichal], [gabbifish], [issue/339][pull/405]**
+
+    Wrangler 1.3.0 includes commands for managing and updating [Workers KV](https://www.cloudflare.com/products/workers-kv/) namespaces, keys, and values directly from the CLI.
+
+    - `wrangler kv:namespace`
+
+      `wrangler kv:namespace` allows developers to `create`, `list`, and `delete` KV namespaces, from the CLI. This allows Wrangler users to configure new namespaces without having to navigate into the Cloudflare Web UI to manage namespaces. Once a namespace has been created, Wrangler will even give you the exact code to copy into your `wrangler.toml` to begin using your new namespace in your project. Neat!
```
Comment (ashleymichal) [inline CHANGELOG.md]: nit
```suggestion
      `wrangler kv:namespace` allows developers to `create`, `list`, and `delete` KV namespaces, from the CLI. This allows Wrangler users to configure new namespaces without having to navigate into the Cloudflare Web UI to manage namespaces. Once a namespace has been created, Wrangler will even give you the exact configuration to copy into your `wrangler.toml` to begin using your new namespace in your project. Neat!
```

**Pull request merged**